### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by uday-nv-work**
Added Cart page

### CODE REVIEW

File: components/Cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.cart-items {
+	width: 100%;
+	padding-bottom: 3rem;
+}
+.cart-header {
+	text-align: center;
+	width: inherit;
+	padding: 25px;
```
Comment (devAmlan) [inline components/Cart/cart.css]: you can use rem here.

File: components/Cart/cart.html [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+			rel="stylesheet"
+			href="https://xero-cl-netlify.netlify.app/css/component.css"
+		/>
+		<link rel="stylesheet" href="/css/component.css" />
+	</head>
+	<body>
+		<div class="component-display-container nav-bar">
+			<div class="nav-container">
+				<div class="nav-brand"
+					><img class="logo" src="/asssets/logo.svg" alt="logo" />
```
Comment (devAmlan) [inline components/Cart/cart.html:22]: your logo image is not showing on ur preview site, so plz check the path.

**Review by devAmlan — APPROVED** [review]

File: css/style.css [hunk 3]
```diff
@@ -70,3 +70,11 @@
 .card-change:hover {
 	transform: scale(1.5);
 }
+.inline {
+	width: 100%;
+	display: inline-flex;
+	/* align-items: flex-end; */
+}
```
Comment (Shubhamkr2610) [inline css/style.css]: Suggestion:remove unused code

**Review by Shubhamkr2610 — APPROVED** [review]
Made the correction that I have suggested. Rest looks good to me. You are good to go after the changes

**Pull request merged**